source/Ship.cpp

```diff
@@ -202,6 +202,16 @@ Ship::Ship(const DataNode &node)
 
 
 
+Ship::Ship(const Ship &ship)
+{
+	*this = ship;
+	// We need to make sure these point to this ship and not the stock model.
+	navigation = ShipJumpNavigation(*this);
+	AICache = ShipAICache(*this);
+}
```

Comment:
This is not very ideal. Wouldn't it be better to make a new function say `Clone` that returns a copy? There would still be the danger of copying a ship without calling that function, but it's probably fine cause ships aren't copied in a lot of places.

Reply:
I'd just get this in as a hotfix tbh
I dont think a clone function is a good idea, either